frontend/src/Frontend/Foundation.hs

```diff
@@ -93,3 +95,14 @@ type family ReflexValue (f :: * -> *) x where
     ReflexValue (Behavior t) x = Behavior t x
 
     ReflexValue (Event t) x = Event t x
+
+addDecimalToString :: Text -> Text
+addDecimalToString s =
+    case T.find f s of
+      Nothing -> s <> ".0"
+      Just _ -> s
+  where
+    f '.' = True
+    f 'e' = True
+    -- Need to also check for 'e' because we don't want "1e-7" to be converted
+    -- to "1e-7.0"
```

Comment:
This looks okay, but there's a [parser for this in the `scientific` library](https://hackage.haskell.org/package/scientific-0.3.6.2/docs/Data-Scientific.html) which does it in a really clean way which could save us the trouble of writing the logic ourselves: 

```haskell
> import Text.ParserCombinators.ReadP (readP_to_S)
> readP_to_S scientificP "3"
[(3.0,"")]
> readP_to_S scientificP "3.0e2"
[(3.0,"e2"),(300.0,"")]
> readP_to_S scientificP "+3.0e+2"
[(3.0,"e+2"),(300.0,"")]
> readP_to_S scientificP "-3.0e-2"
[(-3.0,"e-2"),(-3.0e-2,"")]
```

Reply:
Nice idea, I put that in.